utils/support-images/dpdk-19.11/Dockerfile

```diff
@@ -7,7 +7,7 @@ ENV RTE_SDK=${DPDK_DIR}
 
 # Install prerequisite packages
 RUN dnf groupinstall -y "Development Tools" && \
-    dnf install --skip-broken -y wget numactl numactl-devel make libibverbs-devel logrotate rdma-core tcpdump && \
+    dnf install --skip-broken -y wget numactl numactl-devel make libibverbs-devel logrotate rdma-core tcpdump sudo && \
```

Comment:
question related to this file: Why we keep a Dockerfile for DPDK 19.11 and 23.11? using the latest shouldn't be enough?

Reply:
ack, appreciate the time to clarify.